sw/device/silicon_creator/lib/drivers/ibex.c

```diff
@@ -51,6 +52,13 @@ void ibex_addr_remap_1_set(uint32_t matching_addr, uint32_t remap_addr,
   icache_invalidate();
 }
 
+void ibex_addr_remap_lockdown(uint32_t index) {
+  HARDENED_CHECK_LT(index, 2);
+  index *= sizeof(uint32_t);
+  abs_mmio_write32(kBase + RV_CORE_IBEX_IBUS_REGWEN_0_REG_OFFSET + index, 0);
+  abs_mmio_write32(kBase + RV_CORE_IBEX_DBUS_REGWEN_0_REG_OFFSET + index, 0);
```

Comment:
Can you read back the values to make sure they were written? It may be worth doing the read back check as a separate inline call so that we can call `ibex_addr_remap_lockdown` before `rom_ext_sram_exec()` in the rom followed by the read-back check. This is to deter against faults aiming to skip this function call.

Reply:
Done:  I used `sec_mmio` to lock out the translation windows and then checked values before jumping to owner code.